src/plugins/console/api_server/es_5_0/aggregations.js

```diff
@@ -0,0 +1,436 @@
+var simple_metric = {
+  __template: {field: ""},
+  field: "{field}",
+  missing: 0,
+  script: {
+    // populated by a global rule
+  }
+}, field_metric = {
```

Comment:
Thanks @panda01. I think the entire Console codebase needs to be brought up to speed with the Kibana style guide, now that Console is part of Kibana core. I would prefer to do that outside this PR to keep the changes here scoped to making Console work with Elasticsearch 5.0 APIs. What do you think?